docs/configure-and-manage/configure/consensus-protocols/raft.md

```diff
@@ -8,6 +8,11 @@ GoQuorum implements the Raft Proof-of-Authority (PoA) consensus protocol.
 To enable Raft consensus, specify the [`--raft`](../../../reference/cli-syntax.md#raft) command line option when starting GoQuorum.
 You can [create a private network using Raft](../../../tutorials/private-network/create-a-raft-network.md).
 
+!!! warning
+
+    Raft is not suitable for production environments. Use only in development environments. Also, you can't migrate to
+    another consensus protocol from Raft.
```

Comment:
We need to add a bit more nuance to the migration point: 

Migrating from a network from Raft to a consensus mechanism suitable for production such as QBFT requires one of: 
* Stopping the Raft network and starting the new network with the state at the time of migration.  That is, historical transactions and state history are lost. 
* Replying the historical transactions on the new network.  The historical transactions will be a different block heights but other than that, the transactions and state history will be the same on the new network as on the Raft network. 

If you want Enterprise support to migrate a Raft network, contact us (link to relevant support contact).